modules/tinymce/CHANGELOG.md

```diff
@@ -29,7 +29,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Allow a media embed element to be correctly resized when using the `media` plugin dialog by converting the media embed to a standalone iframe. #TINY-8714
 - Inline alert in the "Search and Replace" dialog persisted when it wasn't necessary. #TINY-9704
 - Context toolbars displayed the incorrect status for the `advlist` plugin buttons. #TINY-9680
-- On iOS Safari, Hangul (Korean) characters will no longer merge onto the previous line after inserting a newline by pressing Enter. #TINY-9746
+- On iOS Safari, Korean characters will no longer merge onto the previous line after inserting a newline by pressing Enter. #TINY-9746
```

Comment:
I remember this issue only occurs in the latest/ a specific version of iOS Safari

Reply:
```suggestion
- On iOS Safari, Korean characters was merging onto the previous line upon typing after inserting a newline by pressing Enter. #TINY-9746
```

Reply:
The ticket reports the version where the issue is reproduced, but we are not sure if the issue persists for iOS Safari <14. Also, specifying specific version numbers in the changelog denotes that the fix was targeted for these versions, but our fix targets all versions iOS Safari.